### PR TITLE
eslint and packages updates

### PR DESCRIPTION
**Pull request by jsnoble**

### CODE REVIEW

File: eslint.config.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import eslintConfig from '@terascope/eslint-config';
+// need to probably put this in original eslint-config
+// don't lint the dist folder
+eslintConfig[0].ignores.push('dist/');
```
Comment (busma13) [inline eslint.config.js:4]: The asset has a dist folder too.
```suggestion
eslintConfig[0].ignores.push('dist/');
eslintConfig[0].ignores.push('asset/dist/');
```

File: .eslintrc [hunk 2]
```diff
@@ -1,14 +0,0 @@
...
-    "extends": "@terascope",
-    "parserOptions": {
-        "ecmaVersion": "latest",
-        "sourceType": "module"
-    },
-    "rules": {
-        "@typescript-eslint/naming-convention": "off",
-        "@typescript-eslint/no-duplicate-enum-values": "warn",
-        "import/extensions": "off",
-        "import/no-import-module-exports": "off"
```
Comment (busma13) [inline .eslintrc:11]: Are we choosing to use the defaults for `no-duplicate-enum-values` and `no-import-module-exports` or are these handled somewhere else?

**Review by busma13 — CHANGES_REQUESTED** [review]

File: .eslintrc (hunk 2 above)
Comment (jsnoble) [reply .eslintrc:11]: these are handled in the eslin-config library, remember these are just overrides, we have not touched the library for a very long time

File: eslint.config.js (hunk 1 above)
Comment (jsnoble) [reply eslint.config.js]: I put that in though its not needed for this, the build just happens to keep this in line, as mentioned in the comments, I will be removing these and putting them back in the eslint-config package itself so this is temporary

**Review by busma13 — APPROVED** [review]
LGTM

**Pull request merged**